### DISCUSSION TITLE
佬，你是不是看看你2.8之后的编译？

### DISCUSSION THREAD

**Discussion by ixxmu** [lxhao61/integrated-examples#223] [General]
2.8之后的版本同款配置（2.6正常的caddyfile）为什么报错了，每一个报错信息都显示trojan模块未正确配置！
是不是2.8之后默认是json的配置，就算是caddyfile也是给他转化成json然后运行的，所以报错了！

> Error: loading initial config: loading new config: app module trojan: module not configured

测试了几个你编译的2.8，都是这个问题,我压根就没有配置trojan！

**Comment by lxhao61** [comment]
不知道你在说什么？

**Reply by ixxmu** [reply]
你编译的caddy 2.8 之后的版本，和2.6的配置文件不兼容，2.8提示 trojan 相关错误，2.7的也没有问题：

**Reply by lxhao61** [reply]
晕！见 Releases 中注意事项，Caddy 及插件升级后的配置本来就有改动！
